lang/fr.js

```diff
@@ -2,24 +2,24 @@
 
 export default {
 	"components.alert.close": "Fermer l'alerte",
-	"components.calendar.notSelected": "Pas sélectionné",
-	"components.calendar.selected": "Sélectionné",
+	"components.calendar.notSelected": "Non sélectionné(e)",
+	"components.calendar.selected": "Sélectionné(e).",
 	"components.calendar.show": "Afficher {month}",
 	"components.dialog.close": "Fermer cette boîte de dialogue",
 	"components.input-date.clear": "Effacer",
-	"components.input-date.openInstructions": "",
-	"components.input-date.setToToday": "",
+	"components.input-date.openInstructions": "Utiliser le format de date {format}. Utiliser la flèche vers le bas ou la touche d’entrée pour accéder au mini-calendrier.",
+	"components.input-date.setToToday": "Régler à aujourd’hui",
 	"components.input-date-time.date": "Date",
-	"components.input-date-time.time": "Heure",
+	"components.input-date-time.time": "Durée",
```

Comment:
I'm going to switch this back, I think "Heure" is more the spirit of what this should be.
```suggestion
	"components.input-date-time.time": "Heure",
```